psh/tools/psh.py

```diff
@@ -285,7 +285,9 @@ def run(harness):
     def wrapper_harness(*args, **kwargs):
         # to avoid CI fails caused by https://github.com/phoenix-rtos/phoenix-rtos-project/issues/866
         if trunner.ctx.target.name == "armv7m7-imxrt117x-evk":
```

Comment:
And then we can remove such line below.
```suggestion
        dut = args[0]
        if trunner.ctx.target.name == "armv7m7-imxrt117x-evk":
```